.babelrc

```diff
@@ -1,4 +1,14 @@
 {
-  "presets": ["es2015", "react", "stage-2"],
-  "plugins": ["lodash"]
+  "presets": ["es2015-webpack", "react", "stage-2"],
+  "env": {
+    "development": {
+      "plugins": ["lodash", "transform-es2015-modules-commonjs"]
+    },
+    "production": {
+      "plugins": ["lodash", "transform-es2015-modules-commonjs"]
+    },
+    "es": {
+      "plugins": ["lodash", "./babel-lodash-es"]
```

Comment:
Sucks to have to duplicate config, but babel always *adds* plugins apparently, and the order is important for the lodash-es plugin. I did both `development` and `production` so you don't have to remember to set `BABEL_ENV` all the time for non-es builds. All `es2015-webpack` does is load `es2015` and remove the `transform-es2015-modules-commonjs` plugin.